Services/Contact/classes/class.ilContactGUI.php

```diff
@@ -128,6 +128,13 @@ public function executeCommand()
 				$this->ctrl->setReturn($this, "showContacts");
 				$this->ctrl->forwardCommand(new ilMailSearchGroupsGUI());
 				break;
+
+			case 'ilmailsearchlearningsequencegui':
```

Comment:
I do not find a class `ilMailSearchLearningSequenceGUI`. I think these changes here are not necessary.

Reply:
Thx. @nhaagen, please have a look.